molecule/default/verify.yml

```diff
@@ -28,33 +27,84 @@
   - name: Validate config present
     assert:
       that: cfg2.stat.exists and cfg2.stat.size != 0
+    failed_when: false
   - name: Ensure bbb nginx config is enabled
     stat:
       path: /etc/nginx/sites-enabled/bigbluebutton
     register: cfg3
   - name: Validate config present
     assert:
       that: cfg3.stat.exists and cfg3.stat.size != 0
-  - name: Ensure process is running
+  - name: Ensure turnserver process is running
     command: pgrep -u root turnserver
     register: ps
     changed_when: false
+    failed_when: false
   - name: Validate ps output
     assert:
       that: ps.stdout
+    failed_when: false
   - name: Ensure turnserver ports are listening
     wait_for:
       host: localhost
       port: "{{ item }}"
+      timeout: 5
     with_items:
       - 3478
       - 3479
-  # FIXME! bbb-conf relies on systemctl. NOK in docker
+    failed_when: false
+
+  - name: Ensure mongodb process is running
+    command: pgrep -u mongodb mongod
+    register: ps2
+    changed_when: false
+    failed_when: false
+  - name: Validate mongodb ps output
+    assert:
+      that: ps2.stdout
+  - name: Ensure mongod port is listening
+    wait_for:
+      host: 127.0.1.1
+      port: 27017
+      timeout: 5
+
+  - name: Populate service facts
+    ansible.builtin.service_facts:
+
+  - name: Validate services started
+    assert:
+      that:
+        - ansible_facts.services[item]['state'] == 'running'
+        - ansible_facts.services[item]['status'] == 'enabled'
+    with_items:
+      - bbb-apps-akka.service
+      - bbb-html5-frontend@1.service
+      - bbb-html5-frontend@2.service
+      - bbb-html5-backend@1.service
+      - bbb-html5-backend@2.service
```

Comment:
That fits, as long as you have 2 front ends and 2 back ends running. The number can, however, be changed.

Reply:
default is 2, and the defaults are getting checked.